desc/objectives/_coils.py

```diff
@@ -1267,6 +1283,197 @@ def compute(self, field_params, constants=None):
         return f
 
 
+class QuadraticFluxMinimizingSurface(_Objective):
```

Comment:
this name makes it sound like this itself is a surface, not an objective. 

Maybe `SurfaceQuadraticFlux` or something?